<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Lubrication</class>
 <widget class="QWidget" name="Main_Lubrication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
	border:1px solid lightgray;
    color:white;
font:16px
}

QPushButton
{
	font:20px;
    color: rgb(240,240,240);
	background-color: rgb(0, 0, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
   border-right: 1px solid white;
   border-bottom: 1px solid white;
}</string>
  </property>
  <widget class="QLabel" name="lbl_information1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>返回0秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_information2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/Cheti.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_information3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/Cheti-reverse.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BP_Running">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>运行测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BP_Cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
